e2e/tests/init_test.go

```diff
@@ -566,6 +589,50 @@ func (s *suite) validateAmountChange(prevAmount, currAmount int, expectedAmountC
 
 }
 
+// getSupplierInfo returns the supplier information for a given supplier address
+func (s *suite) getSupplierInfo(supplierAddr string) *sharedtypes.Supplier {
+	args := []string{
+		"query",
+		"supplier",
+		"show-supplier",
+		accNameToAddrMap[supplierAddr],
+		"--output=json",
+	}
+
+	res, err := s.pocketd.RunCommandOnHostWithRetry("", numQueryRetries, args...)
+	require.NoError(s, err, "error getting supplier %s", supplierAddr)
+	s.pocketd.result = res
+
+	var resp suppliertypes.QueryGetSupplierResponse
+	responseBz := []byte(strings.TrimSpace(res.Stdout))
+	s.cdc.MustUnmarshalJSON(responseBz, &resp)
+	return &resp.Supplier
+}
+
+// getSupplierUnbondingHeight returns the height at which the supplier will be unbonded.
+func (s *suite) getSupplierUnbondingHeight(accName string) int64 {
+	supplier := s.getSupplierInfo(accName)
+
+	args := []string{
+		"query",
+		"shared",
+		"params",
+		"--output=json",
+	}
+
+	res, err := s.pocketd.RunCommandOnHostWithRetry("", numQueryRetries, args...)
+	require.NoError(s, err, "error getting supplier module params")
+
+	var resp sharedtypes.QueryParamsResponse
+	responseBz := []byte(strings.TrimSpace(res.Stdout))
+	s.cdc.MustUnmarshalJSON(responseBz, &resp)
+	windowCloseHeight := shared.GetProofWindowCloseHeight(
+		&resp.Params,
+		int64(supplier.UnstakeCommitSessionEndHeight),
+	)
```

Comment:
~~Wdyt about moving this into a new static function like `shared.GetUnbondedHeight()` or something?~~

Now seeing the keeper method, left a comment there.